github/repos.go

```diff
@@ -284,6 +287,7 @@ func (s *RepositoriesService) Get(owner, repo string) (*Repository, *Response, e
 	// TODO: remove custom Accept header when the license support fully launches
 	// https://developer.github.com/v3/licenses/#get-a-repositorys-license
 	req.Header.Set("Accept", mediaTypeLicensesPreview)
+	req.Header.Add("Accept", mediaTypeSquashPreview)
```

Comment:
Has anyone tested that multiple `Accept` headers actually work by combining both preview APIs, instead of a random one being used?

I'm asking because this is the very first time I see it done in this library, it has never been done before as far as I know.

I tried reading https://developer.github.com/v3/media/ and it doesn't seem to say that multiple Accept headers are the way to go (neither does it say it shouldn't be done).